rfcs/endpoints-needed.md

```diff
@@ -0,0 +1,43 @@
+# Enpoints Needed For Each Page
+
+### Sign In Page
+  - POST /user/login
+
+  #### Sign Up Page  
+
+  - POST /user/signup
+
+  #### Forgot Password Page
+
+  - GET /user/email
+  - POST /user/email/forgotpassword
+
+### Match List Page
+ - GET /matches/{id=userId}
+    * (Score, usernames, logos, team names)
+ - GET /nflweek
+ 
+### Match Details Page
+  - GET /matches/{id=matchId}
+    * (Score, usernames, logos, team names)
+  - GET /fantasyteam/{id=fantasyteamId}
+    * (List of players selected)
+
+### Create Match Page
+
+  - POST /matches/
+  - POST /inventation/{id=matchId}
+  - GET /nflweek
+
+### Player List Page
+
+  - GET /fantasyteam/{id=fantasyTeamId}
+  - GET /players/
+    * (Exclude players already on fantasy team)
+  - GET /players/{search=""}
+  - GET /players/{position=""}
+  - POST /fantasyteam/{playerId=playerId}
```

Comment:
Not sure, but I think we'll probably need to pass in the fantasy team ID too when we post and delete a player from the fantasy team

Reply:
Oh yeah good catch